### ISSUE TITLE
Add support for MI v2 in MIService

### ISSUE THREAD

**Issue by Anders-Karlsson** [infor-cloud/m3-h5-sdk#192]
**Is your feature request related to a problem? Please describe.**
We would like to use MI v2 in our H5 SDK but the MIService is hard coded to use "v1"

**Describe the solution you'd like**
Add support for MI v2 in the MI service

**Comment by MohanabalanD** [comment]
Yes We need support to use v2 since currently odin is supporting only v1 API
